core/tests/protractor_utils/forms.js

```diff
@@ -537,7 +583,10 @@ var RichTextChecker = async function(arrayOfElems, arrayOfTexts, fullText) {
       for (var i = 1; i < arguments.length; i++) {
         args.push(arguments[i]);
       }
-      expect(await elem.getText()).toBe(arrayOfTexts[arrayPointer]);
+      expect(
+        await action.getText(
+          `${elem} in Array of Elements in Rich Text Area`, elem))
+        .toBe(arrayOfTexts[arrayPointer]);
```

Comment:
The current indentation makes it hard to keep track of which parentheses go together. This might be clearer:

```suggestion
      expect(
        await action.getText(
          `${elem} in Array of Elements in Rich Text Area`, elem)
      ).toBe(arrayOfTexts[arrayPointer]);
```